frontend/packages/operator-lifecycle-manager/src/components/operand/create-operand.tsx

```diff
@@ -121,8 +126,8 @@ export const CreateOperand: React.FC<CreateOperandProps> = ({
   );
 
   return (
-    <StatusBox loaded={loaded} loadError={loadError} data={clusterServiceVersion}>
-      {loaded ? (
+    <StatusBox loaded={loaded} loadError={loadError} data={csv}>
+      {loaded && (
         <>
```

Comment:
No need to conditionally render on `loaded`, StatusBox takes care of that. I think the React fragment wrapper can be removed too.